apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -374,8 +406,34 @@ const TutorialExplorer = React.createClass({
      */
     findMatchingTag(filterGroup, tutorialTags) {
       return filterGroup.some(filterName => tutorialTags.split(',').indexOf(filterName) !== -1);
-    }
+    },
 
+    /* Returns an array of unique organization names from the set of tutorials,
+     * sorted alphabetically, truncated and ellipsed.
+     *
+     * @param {Array} tutorials - Array of tutorials.
+     * @return {Array} - Array of strings.
+     */
+    getUniqueOrgNamesFromTutorials(tutorials) {
+      // Filter out tutorials with do-not-show tag.
+      const availableTutorials = tutorials.filter(t => {
+        return t.tags.split(',').indexOf("do-not-show") === -1 && t.orgname !== "do-not-show";
```

Comment:
Should we have a const somewhere for `do-not-show` along with a comment explaining what it is/how it's used?

Reply:
Good idea.